seqr/views/apis/users_api.py

```diff
@@ -28,20 +28,25 @@ def __init__(self, error, status_code=400, existing_user=None):
 
 
 @login_required(login_url=API_LOGIN_REQUIRED_URL)
-def get_all_collaborators(request):
+def get_all_collaborator_options(request):
     if request.user.is_staff:
-        collaborators = {user.username: _get_json_for_user(user) for user in User.objects.exclude(email='')}
+        collaborators = User.objects.exclude(email='').filter(is_active=True)
     else:
-        collaborators = {}
+        collaborators = set()
         for project in get_projects_user_can_view(request.user):
-            collaborators.update(get_project_collaborators_by_username(request.user, project, include_permissions=False))
+            collaborators.update(project.get_collaborators())
 
-    return create_json_response(collaborators)
+    return create_json_response({
+        user.username: _get_json_for_user(user, fields=USER_OPTION_FIELDS) for user in collaborators
+    })
 
 
 @login_required(login_url=API_LOGIN_REQUIRED_URL)
-def get_all_staff(request):
-    staff_analysts = {staff.username: _get_json_for_user(staff) for staff in User.objects.filter(is_staff=True)}
+def get_all_staff_options(request):
+    staff_analysts = {
+        staff.username: _get_json_for_user(staff, fields=USER_OPTION_FIELDS)
```

Comment:
Adding `is_anvil=False` can save Terra API accesses.

Reply:
since `USER_OPTION_FIELDS` does not include `is_anvil`, anvil isn't checked